<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResumeGenerator</class>
 <widget class="QMainWindow" name="ResumeGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resume Generator</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">selection-background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.19397 rgba(0, 0, 0, 255), stop:0.202312 rgba(122, 97, 0, 255), stop:0.495514 rgba(76, 58, 0, 255), stop:0.504819 rgba(255, 255, 255, 255), stop:0.79 rgba(255, 255, 255, 255), stop:1 rgba(255, 158, 158, 255));</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <layout class="QGridLayout" name="previewLayout">
      <item row="1" column="0">
       <widget class="QTextBrowser" name="previewBrowser">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelPreview">
        <property name="text">
         <string>Resume Preview</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="17" column="2" colspan="2">
       <widget class="QTextEdit" name="inputSummary">
        <property name="placeholderText">
         <string>Objective</string>
        </property>
       </widget>
      </item>
      <item row="18" column="2">
       <widget class="QPushButton" name="buttonaddExperience">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="27" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Certification/Achievement</string>
        </property>
       </widget>
      </item>
      <item row="25" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="25" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="16" column="2" colspan="2">
       <widget class="QLineEdit" name="inputGithub">
        <property name="placeholderText">
         <string>.com</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="26" column="1" colspan="3">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelPhone">
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QLineEdit" name="inputEmail">
        <property name="placeholderText">
         <string>@gmail.com</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLineEdit" name="inputLinkedin">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>linkedin.com</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="21" column="1" colspan="3">
       <widget class="QTableView" name="tableSkills"/>
      </item>
      <item row="16" column="1">
       <widget class="QLabel" name="labelGithub">
        <property name="text">
         <string>Website/Github</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="inputLocation">
        <property name="maximumSize">
         <size>
          <width>264</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>State/City</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QLabel" name="labelSummary">
        <property name="text">
         <string>Summary</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="inputPhone">
        <property name="placeholderText">
         <string>+91</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Full Name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="23" column="1" colspan="3">
       <widget class="QTableView" name="tableView_3"/>
      </item>
      <item row="28" column="1" colspan="3">
       <widget class="QTableView" name="tableView_2"/>
      </item>
      <item row="20" column="2">
       <widget class="QPushButton" name="buttonaddSkills">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelLinkedin">
        <property name="text">
         <string>linkedin</string>
        </property>
       </widget>
      </item>
      <item row="27" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="22" column="3">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="27" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <widget class="QLabel" name="labelSkills">
        <property name="text">
         <string>Skills</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QLabel" name="labelExperience">
        <property name="text">
         <string>Work Experience:</string>
        </property>
       </widget>
      </item>
      <item row="22" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Education</string>
        </property>
       </widget>
      </item>
      <item row="25" column="1">
       <widget class="QLabel" name="labelEducation">
        <property name="text">
         <string>Publication</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLabel" name="labelHobies">
        <property name="text">
         <string>Hobies/Interest</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelLocation">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="inputName">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="22" column="2">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1" colspan="3">
       <widget class="QTableView" name="tableExperience"/>
      </item>
      <item row="20" column="3">
       <widget class="QPushButton" name="buttonremoveSkills">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="18" column="3">
       <widget class="QPushButton" name="buttonremoveExperience">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="14" column="2" colspan="2">
       <widget class="QLineEdit" name="inputHobies">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>Generate PDF</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
